source/includes/_events.md

```diff
@@ -1,5 +1,22 @@
 # Events
 
+Every call to action is represented in the API as an event.  Events have related resources such as a contact or a location.
```

Comment:
I also like `ctas` as a resource name, which would make the model `Cta` (which maybe reads a bit weirdly in ruby, but it's not terrible.